lib/caching-precompiler.js

```diff
@@ -1,142 +1,180 @@
 'use strict';
-var fs = require('fs');
 var path = require('path');
+var fs = require('fs');
+var generateMapFileComment = require('convert-source-map').generateMapFileComment;
+var convertSourceMap = require('convert-source-map');
```

Comment:
@novemberborn Yes, https://github.com/jfmengels/eslint-plugin-import-order/blob/master/docs/rules/import-order.md, which will soon be included in XO.